src/App.tsx

```diff
@@ -1,25 +1,26 @@
-import React from 'react';
-import './App.css';
-import { Row, Col } from 'antd';
-import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';
-import MainNavigation from './Components/navigation/MainNavigation';
-import SidebarNavigation from './Components/navigation/SidebarNavigation';
+import React from "react";
+import "./App.css";
+import { Row, Col } from "antd";
+import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
+import MainNavigation from "./Components/navigation/MainNavigation";
+import SidebarNavigation from "./Components/navigation/SidebarNavigation";
 import SendInvitation, {
-  menu as sendInvitationMenu,
-} from 'Pages/SendInvitation';
-import ManageFriend, { menu as manageFriendMenu } from 'Pages/ManageFriend';
-
+  menu as sendInvitationMenu
+} from "Pages/SendInvitation";
+import ManageFriend, { menu as manageFriendMenu } from "Pages/ManageFriend";
+import { Provider } from "react-redux";
+import store from "./Store/Store";
 const contentStyle = {
-  paddingLeft: '10px',
-  textAlign: 'left',
-  overflow: 'auto',
-  height: 'auto',
-  minHeight: '400px',
+  paddingLeft: "10px",
+  textAlign: "left",
+  overflow: "auto",
+  height: "auto",
+  minHeight: "400px"
 } as React.CSSProperties;
 
 const navigationStyle = {
-  width: '1024px',
-  margin: '0 auto',
+  width: "1024px",
+  margin: "0 auto"
```

Comment:
Lets use single quote for convention as we already have single quote as convention.

Reply:
Updated :+1: